verify_backend_features.py

```
@@ -0,0 +1,52 @@
+import requests
+import json
+
+BASE_URL = "http://localhost:8000"
+
+def test_text_category():
+    print("\nTesting Text Category Detection...")
+    text = "There is a large pothole on Main Street causing traffic."
+    try:
+        response = requests.post(f"{BASE_URL}/api/detect-category-text", json={"text": text})
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)